sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -203,6 +203,12 @@ export class StreamingReceiver extends MessageReceiver {
           sbError,
           `${this.logPrefix} 'receiver_error' event occurred. The associated error is`
         );
+        this._messageHandlers().processError({
```

Comment:
Shouldnt we also do this on `_onSessionError`?

Reply:
It was removed too in the same PR https://github.com/Azure/azure-sdk-for-js/pull/11973, I will add it back. The original code has a check `if (receiver && !receiver.isItselfClosed())` that I could add back too, although my testing shows that it doesn't make any difference.  Once the user call close() on the subscription, we no longer retry, so no more _onAmqpError() calls.

Reply:
I agree, we don't need the isItselfClosed check